packages/copyable/src/Copyable.tsx

```diff
@@ -160,23 +199,31 @@ interface CopyableProps {
 }
 
 export default function Copyable({
-  darkMode = false,
+  darkMode: darkModeProp,
   children,
   label,
   description,
   className,
   copyable = true,
-  size = Size.Default,
+  size,
   shouldTooltipUsePortal = true,
 }: CopyableProps) {
-  const mode = darkMode ? Mode.Dark : Mode.Light;
-
+  const { theme, darkMode } = useDarkMode(darkModeProp);
   const [copied, setCopied] = useState(false);
   const [showCopyButton, setShowCopyButton] = useState(false);
   const [buttonRef, setButtonRef] = useState<HTMLButtonElement>();
 
   const { portalContainer } = usePopoverPortalContainer();
 
+  // TODO: should API change to use number values instead of 'large and default' for component size?
```

Comment:
I think keeping the API consistent here with other components (i.e. `small | default | large`) makes sense, but I agree that we should have a utility that computes the appropriate baseFontSize based on the size prop